src/libcrun/cgroup.c

```diff
@@ -890,8 +956,7 @@ int enter_systemd_cgroup_scope (runtime_spec_schema_config_linux_resources *reso
   int sd_err, ret = 0;
   sd_bus_error error = SD_BUS_ERROR_NULL;
   const char *object;
-  int terminated = 0;
-  struct systemd_job_removed_s userdata;
+  struct systemd_job_removed_s status;
```

Comment:
we need to memset the struct to zero, or make sure the pointers are initialized to `NULL`, otherwise we might end up to `free()` garbage values

Reply:
also, could we move it under `systemd_check_job_status_setup` since it seems to not be used here anymore?

Reply:
> we need to memset the struct to zero

sorry, too much golang recently :-\

> could we move it under `systemd_check_job_status_setup` 

Something i was thinking about, too, after looking into systemd code.

done